src/lib/credentials.ts

```diff
@@ -17,9 +17,6 @@ export const GOV_UK_CREDENTIAL = "https://vocab.account.gov.uk/GovUKCredential";
 export const GOV_UK_HAS_CREDENTIAL =
   "https://vocab.account.gov.uk/hasCredential";
 
-// eslint-disable-next-line no-shadow
-import { Blob } from "node:buffer";
```

Comment:
Is this a duplicate `Blob`? I thought we had to import this as per your comment [here](https://github.com/alphagov/di-solid-prototype/blob/335f40a648ade128606d78379835c4a6b8eda957/src/lib/pod.ts#L17-L33)

Reply:
Ah sorry perhaps it wasn't duplicate then 🤔 this might have been the Eslint rule ordering one.

Reply:
https://github.com/alphagov/di-solid-prototype/pull/90/commits/a48cd6e88ba2bedd612ccdef99f380cff2ea246d

Added this!